vivisect/parsers/__init__.py

```diff
@@ -11,12 +11,20 @@
 """
 # Some parser utilities
 
+import imp
 import md5
 import sys
+import glob
 import struct
+import hashlib
+import importlib
 
 import vstruct.defs.macho as vs_macho
 
+from os.path import dirname, basename, isfile, join
+parserpaths = glob.glob(join(dirname(__file__), "*.py"))
+__all__ = [ basename(f)[:-3] for f in parserpaths if isfile(f) and not f.endswith('__init__.py')]
```

Comment:
imho, i dont like this. can we instead do explicit imports and avoid runtime resolution of source code?